### DISCUSSION TITLE
How to keep formatting when copying to external Application? (Teams, Outlook, ...)

### DISCUSSION THREAD

**Discussion by marcosgcd** [singerdmx/flutter-quill#2354] [General]
With example from https://quilljs.com/docs/formats it works fine but I can't get it to work on flutter 

### Quill JS
<img width="836" alt="Bildschirmfoto 2024-11-07 um 14 37 04" src="https://github.com/user-attachments/assets/0194093c-a1ca-4bca-ac7b-fc7528cc98f2">
<img width="613" alt="Bildschirmfoto 2024-11-07 um 14 36 25" src="https://github.com/user-attachments/assets/b45f93b5-d026-4a9c-88b3-6f4f3b3c1f93">

### Flutter Quill
<img width="777" alt="Bildschirmfoto 2024-11-07 um 14 39 52" src="https://github.com/user-attachments/assets/6c2ee21d-69e4-4c7a-a8ff-502c9285c93c">
<img width="615" alt="Bildschirmfoto 2024-11-07 um 14 40 03" src="https://github.com/user-attachments/assets/26fa994a-41c8-4fd9-952f-67d8f775aa48">

**Comment by EchoEllet** [comment]
You will need to convert the `Delta` to HTML and copy that HTML to the system clipboard, which involves platform-specific API, you can use:

* [`super_clipboard`](https://pub.dev/packages/super_clipboard) or [`quill_native_bridge`](https://pub.dev/packages/quill_native_bridge) (an internal plugin) for writing to the system clipboard or using platform code with Flutter method channel.
*  [`vsc_quill_delta_to_html`](https://pub.dev/packages/vsc_quill_delta_to_html) for converting the **Delta** to **HTML**.

However, currently, the project doesn't provide a way to customize the copy behavior, this might be fixed in #2338 though unlikely.
